src/modules/interactive.js

```diff
@@ -0,0 +1,13 @@
+import { printDynamic, eachList } from './Add&Remove.js';
+
+const noCompleted = () => {
+  const comptdTaskList = eachList.filter((noCompletedTask) => noCompletedTask.completed === false);
+  printDynamic(comptdTaskList);
+  localStorage.clear();
+  localStorage.setItem('tasks', JSON.stringify(comptdTaskList));
+  // Code below for future improvements and Testing purposes only
+  // console.log(comptdTaskList);
```

Comment:
It is better to remove unnecessary comments

Reply:
Thank you for your suggestion, was helpful. I already resolved that.